### PR TITLE
Create index.md in /criteria/ for da translation

### PR DESCRIPTION
**Pull request by jgroenen**

### CODE REVIEW

**Comment by jgroenen** [comment]
This fixes #61

File: da/criteria/index.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Kriterier
+
+{% assign sorted = site.pages | sort:"order" %}
+
+{% for page in sorted %}{% if page.dir == "/es/criteria/" %}{% if page.name != "index.md" %}{% if page.title %}
```
Comment (clausmullie) [inline da/criteria/index.md:5]:
```suggestion
{% for page in sorted %}{% if page.dir == "/da/criteria/" %}{% if page.name != "index.md" %}{% if page.title %}
```

**Review by clausmullie — APPROVED** [review]
Tested this locally - it works!

**Review by Ainali — APPROVED** [review]
Great!

**Pull request merged**